### DISCUSSION TITLE
spec file is missing,I can't find it.

### DISCUSSION THREAD

**Discussion by freshstart1524** [Auties00/WhatsappWeb4j#440] [Q&A] [answered]
<img width="1100" alt="aa5abac3fc9a1cdfb5552d66ebb17b5" src="https://github.com/Auties00/Cobalt/assets/57868817/cc553d5f-1e43-47ac-863b-1728b46739ed">
<img width="375" alt="57fe9597291eb5df2d9f2adfb19f842" src="https://github.com/Auties00/Cobalt/assets/57868817/39838213-2b0b-40bf-b3a2-8e21d4114797">

the question is I can't find the spec file in source code, but it exists in the jar

**Comment by Auties00** [comment, marked as answer]
The file is generated at compile time, just compile the project and it will show up in the generated sources

**Reply by freshstart1524** [reply]
thanks, I will try it
